utils/Echarts.ts

```diff
@@ -0,0 +1,79 @@
+import * as echarts from "echarts";
+import { partyColorFromSlug, partyNameFromSlug } from "./Party";
+
+const getChartData = (partyData: PartyVoteResult[]) => {
+  const axisArray: string[] = [];
+  const seriesArray: {
+    value: number;
+    itemStyle: {
+      color: string;
+    };
+  }[] = [];
+
+  for (let i = 0; i < partyData.length; i++) {}
+  partyData.forEach((resultData: PartyVoteResult) => {
+    if (resultData.votePercent > 0) {
+      axisArray.push(partyNameFromSlug(resultData.partySlug));
+
+      if (resultData.votePercent < 15) {
+        seriesArray.push({
+          value: resultData.votePercent,
+          itemStyle: { color: partyColorFromSlug(resultData.partySlug) },
+        });
+      } else {
+        seriesArray.push({
+          value: resultData.votePercent,
+          itemStyle: { color: partyColorFromSlug(resultData.partySlug) },
+        });
+      }
```

Comment:
Am I missing something here? if/else are doing the same thing?

Reply:
Ah yeah I was putting labels on them and formatting them differently for small bars but decided against it.